config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
```

Comment:
- [ **OPTIONAL** ] What do you think if you use the `resources` method to define your routes, as this seems to be easy and cleaner code.
```ruby
  root "users#index"

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```